include/superior_mysqlpp/prepared_statements/initialize_bindings.hpp

```diff
@@ -47,9 +47,9 @@ namespace SuperiorMySqlpp
         constexpr inline void initializeNullable(MYSQL_BIND& binding, Nullable<T>& nullable)
         {
             // Checks if pointer of C client pointer to their custom bool bool variant is compatible with bool*
-            // TODO: It would be wise to check also sizeof(my_bool) == sizeof(bool)
-            static_assert(sizeof(decltype(binding.is_null)) == sizeof(decltype(&nullable.detail_getNullRef())),
-                "Pointers to null indicators must have same size.");
+            static_assert(sizeof(decltype(binding.is_null)) == sizeof(decltype(&nullable.detail_getNullRef()))
+                && sizeof(decltype(*binding.is_null)) == sizeof(decltype(nullable.detail_getNullRef())),
```

Comment:
Is it really needed? Nullable flag is initializable even without second condition, isnt it?

Reply:
`binding.is_null` has type `my_bool*`, &nullable.detail_getNullRef() has type `bool*`, we feed our pointer to C client.
So, we clearly need sanity checks that `my_bool` has same size as `bool`, the second condition. That said, we perhaps do NOT need the first one, as the second one likely implies the first one, that their respective pointers have matching sizes.

Reply:
I'd prefer remove the first part of condition, checking pointer sizes. The second condition make sense to me.

Reply:
After further offline discussion, we ended up keeping both conditions, main argument being that the original condition was done by original lib author, although without commentary what real situation it was meant to cover.